cmd/operator/deploy/operator/11-rule-evaluator.yaml

```diff
@@ -49,11 +49,19 @@ spec:
                 operator: In
                 values:
                 - linux
-      serviceAccountName: collector
```

Comment:
You removed this as it's [already part](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/c4f3239d50e536b60ba730e91fce1f9fa3069c93/cmd/operator/deploy/operator/11-rule-evaluator.yaml#L52) of the template spec, correct?

Reply:
Yes.